### ISSUE TITLE
Progress indicator

### ISSUE THREAD

**Issue by KaelTeck** [7mada123/disposable_cached_images#6]
This package is great but could you add a progress indicator option when loading an image?

**Comment by 7mada123** [comment]
Hi KaelTeck, thank you for your feedback.
I will definitely take it into consideration for the next release version.

**Comment by KaelTeck** [comment]
Thank you very much.

**Comment by 7mada123** [comment]
Progress indicator is now available In version `1.2.0`
```dart
DisposableCachedImage.network(
  ...
  progressBuilder: (context, progress) => Center(
    child: LinearProgressIndicator(value: progress),
  ),
);
```

**Comment by KaelTeck** [comment]
Thank you so much!!!
